specification/http/src/main/scripts/org/kaazing/specification/http/rfc7234/expires/already.expired.conditional.request.304/request.rpt

```diff
@@ -0,0 +1,29 @@
+connect http://localhost:8000/index.html
+connected
+
+write method "GET"
+write version "HTTP/1.1"
+write header host
+write header content-length
+write flush
+
+read status "200" /.*/
+read header "Last-Modified" /(?<lastModified>.+)/
+read header "Expires" /.+/
+read notify EXPIRES
+
+connect await EXPIRES
```

Comment:
Perhaps `LAST_MODIFIED` instead of `EXPIRES` for barrier name since that's the important part here?